app/support/page.tsx

```diff
@@ -0,0 +1,72 @@
+'use client'
+import Link from 'next/link';
+import React from 'react';
+
+const handleSave = (e: { preventDefault: () => void; }) =>{
+  e.preventDefault()
+  console.log("save button clicked!")
+}
+
+
+export default function Support() {
+  return (
+    <form className="max-w-lg mx-auto mt-28 p-8">
+      <div className="space-y-12">
+        <div>
+          <h2 className="text-base/7 font-semibold text-gray-900">Support</h2>
+          <p className="mt-1 text-sm/6 text-gray-600">
+          We’re here to support you every step of the way. Just share your email and let us know what’s on your mind—whether it’s a question, a suggestion, or something you need help with. Our team will be in touch soon to make sure you’re taken care of!
+          </p>
+
+          <div className="mt-10 grid grid-cols-1 gap-x-6 gap-y-8 sm:grid-cols-6">
+          <div className="sm:col-span-4">
+              <label htmlFor="email" className="block text-sm/6 font-medium text-gray-900">
+                Email address
+              </label>
+              <div className="mt-2">
+                <input
```

Comment:
Can we add a similar email validation as the subscription form? the code is in:
https://github.com/Servgenie/servgenie-web-frontend/blob/88fcfca840c8cac1817c12863307adedac18385e/components/section/Home/GetReady.tsx#L37

and if you're up for it, for best practice, we can consolidate the shared validation or code into a utility file and import that utility into the component that needs it.

Step 1: Create a file, for example, formUtils.ts, in a suitable /shared directory like utils or lib.
```
class FormUtils {
  private emailRegex = /^[^\s@]+@[^\s@]+\.[^\s@]+$/;

  /**
   * Validates an email address.
   * @param email - The email to validate.
   * @returns True if the email is valid, otherwise false.
   */
  validateEmail(email: string): boolean {
    return this.emailRegex.test(email);
  }

  /**
   * Executes the recaptcha function and returns the token.
   * @param executeRecaptcha - The recaptcha execution function.
   * @param action - The action to pass to the recaptcha.
   * @returns The recaptcha token or null if failed.
   */
  async verifyCaptcha(
    executeRecaptcha: (() => Promise<string>) | null,
    action: string
  ): Promise<string | null> {
    if (!executeRecaptcha) return null;
    try {
      return await executeRecaptcha(action);
    } catch {
      return null;
    }
  }
}

export default new FormUtils();

```

Step 2: Use the Utility Class in the Components
In your component, import and use the utility class to handle form logic, for example, in the existing handleSubscribe function
```
import React, { useState } from 'react';
import FormUtils from '../utils/formUtils'; // Adjust the path as necessary
import toast from 'react-hot-toast';

const SubscribeForm = ({ executeRecaptcha }: { executeRecaptcha: (() => Promise<string>) | null }) => {
  const [email, setEmail] = useState('');
  const [loading, setLoading] = useState(false);

  const handleSubscribe = async (e: React.FormEvent<HTMLFormElement>) => {
    e.preventDefault();

    // Validate Captcha
    const token = await FormUtils.verifyCaptcha(executeRecaptcha, 'newsletter_subscribe');
    if (!token) {
      toast.error("We've detected a bot-like behavior. Please try again later.");
      return;
    }

    // Validate Email
    if (!FormUtils.validateEmail(email)) {
      toast.error('Please enter a valid email address!');
      return;
    }

    setLoading(true);

    try {
          ....// Subscribe or Save functionality below
    } finally {
      setLoading(false);
    }
  };
```

This benefits the code in terms of:
Reusability: Common logic is reusable across multiple components.
Maintainability: Changes to the logic (e.g., updating regex) only need to be made in one place.